dts/bindings/pinctrl/nuvoton,npcx-pinctrl.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The Nuvoton pin controller is a singleton node responsible for controlling
+  pin function selection and pin properties. For example, you can use these
+  nodes to select peripheral pin functions.
+
+  Here is a list of supported standard pin properties:
+    - bias-pull-down: Enable pull-down resistor.
+    - bias-pull-up: Enable pull-up resistor.
+    - drive-open-drain: Output driver is open-drain.
+
+  Custom pin properties for npcx series are available also:
+    - pinmux-locked: Lock pinmux configuration for peripheral device
+    - pinmux-gpio: Inverse pinmux back to gpio
+    - drive-supported: Specific pads that support drive mode selection
```

Comment:
Non-blocking comment: delete this line because `drive-supported` should not be modified by the user.  You do note this in help below.

Reply:
Done.